reference/interbit-ui-components/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Interbit UI Components
+
+A library of React UI components specifically designed for use with Interbit.
+
+Interbit UI Components is intended to help developers with the frontend UI
```

Comment:
'frontend' is two words.

Reply:
Commonly used variants are front end, front-end, and frontend (and similar for backend). Does @btlewan have a preference for which is used in our docs?
https://english.stackexchange.com/questions/34447/is-it-front-end-frontend-or-front-end

Reply:
My preference is front-end

Reply:
It depends on the use. If you use it as a compound adjective, as in "front-end developer", it should be hyphenated. If used as a noun (kind of), as in "Go to the front end", it's really "front" as an adjective of "end", and so: front end.

Despite the popularity of removing "unnecessary" hyphens and spaces, using such compressed words is a detriment to non-English speakers and makes any translation efforts more difficult. For example, as "frontend", it's less clear where the syllable stresses should be... is it "front" and "end", "fron" and "tend", or "fronten" and "d"?

That said, there are plenty of examples using "frontend", so I didn't question it.